docs/admin/add_namespace_to_cluster.md

```diff
@@ -0,0 +1,26 @@
+# Add namespace to cluster
+* sops 3.6+ 
```

Comment:
Shouldn't this be under a "prerequisites" header?

Reply:
fixed, and amended other instructions in the admin section as well